plasma_framework/test/src/exits/payment/PaymentChallengeIFENotCanonical.test.js

```diff
@@ -60,17 +63,23 @@ contract('PaymentInFlightExitRouter', ([_, ifeOwner, inputOwner, outputOwner, co
             IFE_TX_TYPE, [buildUtxoPos(0, 0, 0)], [output],
         );
 
-        return {
-            inputTx: web3.utils.bytesToHex(inputTx.rlpEncoded()),
-            decodedInputTx: inputTx,
-        };
+        return web3.utils.bytesToHex(inputTx.rlpEncoded());
+    };
+
+    const createDepositInputTransaction = (outputType) => {
+        const output = new PaymentTransactionOutput(
+            outputType, TEST_COMPETING_TX_OUTPUT_AMOUNT, buildOutputGuard(competitorOwner), ETH,
```

Comment:
probably should be `TEST_IFE_INPUT_AMOUNT` instead of `TEST_COMPETING_TX_OUTPUT_AMOUNT`.
also should be `buildOutputGuard(inputOwner)`